internal/contour/endpointstranslator.go

```diff
@@ -34,14 +34,17 @@ import (
 // A EndpointsTranslator translates Kubernetes Endpoints objects into Envoy
 // ClusterLoadAssignment objects.
 type EndpointsTranslator struct {
+	Cond
 	logrus.FieldLogger
-	clusterLoadAssignmentCache
+
+	lock    sync.Mutex
```

Comment:
```suggestion
	mu     sync.Mutex
```

To be consistent with Contour, let's call this `mu`.